docs/documentation/getting-started.md

```diff
@@ -335,93 +341,91 @@ You can then delete the first account, add a new account with the right values,
 
+- Alternatively, select a module in the project view and right-click it.
+- Select "Share With..." from the options (or use the keyboard-shortcut 's').
+- Select a user from the list (or use their number as the keyboard-shortcut).
 
 ### Starting a Session - Client
 
-- Choose the Saros window in the bottom bar.
-- Choose the "Connect" icon (connected plug icon; the left icon).
+- Click the "Connect" button (![connect icon](images/icons-i/connect.png)).
 - Choose an account.
 - Wait until the host invites you to join their session.
 - After the host invited you, the session negotiation will open.
 - Click "Next". This will open the project negotiation.
-- Choose which local module to use for the session. The automatically chosen option and value should be correct in most cases, so you should not have to change anything.
-  - *To create a new module:* Choose "Create new module" if the shared module is not already present in your local project.
-  - *To use an existing module:* Choose "Use existing module" if a version of the shared module is already present in your local project.
-    - Click on "Browse..." and choose the base directory of the shared module.
-    - The base directory has to have the same name as the shared module.
-    - If the field is left empty, the project negotiation is aborted due to a local error.
-- Click "Next". This will show you the local file changes that will be made during the negotiation. These are the differences between the local version of the module and the version held by the host.
-  - The shown actions are the actions necessary to align the local module with the host module.
-  - Any local differences will be removed during the project negotiation. These adjustments will only be done if the "Finish" button is selected. If the session negotiation is aborted at this stage, no local files are changed.
+- Choose which project to use. This can be done through the drop-down menu at the top of the dialog.
+- Choose how to represent the shared module locally. You can either choose an existing module or create a new one
+  - *To create a new module:* Choose "Create new module"
+    - Specify a module name (must not already exist in the shared project).
+    - Specify the module base path.
+  - *To use an existing module:* Choose "Use existing module"
+    - Select a module from the drop-down menu.
+- Click "Next".
+  - If an existing local module was chosen, a list of local file changes that will be made during the negotiation will be shown. These are the differences between the local version of the module and the version held by the host. The shown actions are the actions necessary to align the local module with the host module.
+  - **Warning:** Any local differences will be removed during the project negotiation. These adjustments will only be done if the "Finish" button is selected. If the session negotiation is aborted at this stage, no local files are changed.
 - Click "Finish".
 
+### Changing user colors
```

Comment:
Could be a little bit confusing when one knows how a User color is defined in Eclipse, i.e that is actually the color you want to use during the session.

Reply:
Yeah I know. Also the explanation on how to use the "user color x" is not really helpful. But I didn't know how to explain it better. It currently is just a weird behavior.

Reply:
It might be easier to understand if you write that this property only changes the local visualization. Otherwise, some users might think that this will change the colors which are used in the editors of the session participants.